mopidy/core/library.py

```diff
@@ -236,7 +236,7 @@ def lookup(self, uri=None, uris=None):
                 result = future.get()
                 if result is not None:
                     validation.check_instances(result, models.Track)
-                    results[u] = result
+                    results[u] = [r for r in result if r.uri]
```

Comment:
I'm adding a note about making `Track.uri` mandatory instead. When we do that, backend's will fail to create the broken objects and will thus quickly get feedback and get fixed. Don't want to add too much logic in an inner loop.